manual/src/refman/modtypes.etex

```diff
@@ -262,6 +262,17 @@ No restrictions are placed on the type of the functor argument; in
 particular, a functor may take another functor as argument
 (``higher-order'' functor).
 
+When the result module type is itself a functor, one may write
+\begin{center}
+@'functor' '(' name_1 ':' module-type_1 ')' \ldots
+           '(' name_n ':' module-type_n ')' '->' module-type@
+\end{center}
+instead of
+\begin{center}
+@'functor' '(' name_1 ':' module-type_1 ')' '->' \ldots '->'
+ 'functor' '(' name_n ':' module-type_n ')' '->' module-type@
+\end{center}
```

Comment:
I am wondering if reversing the two code examples would not be slightly simpler:

> When the result module type is itself a functor,
\begin{center}
@'functor' '(' name_1 ':' module-type_1 ')' '->' \ldots '->'
 'functor' '(' name_n ':' module-type_n ')' '->' module-type@
\end{center}
one may use the abbreviated form
\begin{center}
@'functor' '(' name_1 ':' module-type_1 ')' \ldots
           '(' name_n ':' module-type_n ')' '->' module-type@
\end{center}

Like this the expression "When the result module type is itself a functor" is directly illustrated. (This is a nitpick, it is completely fine to ignore it).